dashboards_bundlers/server_upload/__init__.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+import os
+import requests
+from notebook.utils import url_path_join
+from tornado import escape
+from tornado.log import access_log
+from ..local_deploy import bundle_file_references
+
+ENDPOINT = '/notebooks/'
+
+def bundle(handler, abs_nb_path):
+    '''
+    Uploads a notebook to a Jupyter Dashboard Server
+    '''
+    # Get name of notebook from filename
+    notebook_basename = os.path.basename(abs_nb_path)
+    notebook_name = os.path.splitext(notebook_basename)[0]
+    if 'DASHBOARD_SERVER_URL' in os.environ:
+        server_url = url_path_join(os.getenv('DASHBOARD_SERVER_URL'), ENDPOINT, escape.url_escape(notebook_name, False))
```

Comment:
Not sure how stable our API will be. Drew's already changing it to clean it up. Might be we want the URL directly to the notebooks API endpoint. But for now, leave it.